onnxruntime/python/tools/pytorch_export_contrib_ops.py

```diff
@@ -1,21 +1,38 @@
 # Copyright (c) Microsoft Corporation. All rights reserved.
 # Licensed under the MIT License.
-#
-# Register pytorch symbolic for export using ONNX Runtime contrib ops
 
-from torch.onnx import register_custom_op_symbolic
+"""
+Support for registering ONNX Runtime's built-in contrib ops with
+PyTorch-ONNX exporter (torch.onnx.export).
+"""
+
+import typing
+
+try:
+    from torch.onnx import register_custom_op_symbolic
+except ModuleNotFoundError:
+    raise ModuleNotFoundError(
+        "This module is only useful in combination with PyTorch. "
+        "To install PyTorch see https://pytorch.org/.")
 import torch.onnx.symbolic_helper as sym_help
+import torch.onnx.symbolic_registry as sym_registry
 
-_onnx_opset_version = 1
+__OPSET_VERSION = 1
+__registered_ops: typing.AbstractSet[str] = set()
 
 
-def register_custom_op():
-    """
-    This function registers symbolic functions for
-    custom ops that are implemented as part of ONNX Runtime
+def __reg(symbolic_fn: typing.Callable):
```

Comment:
Name is a little vague. We usually use single `_` to refer to private stuff in ORT. Double `_` is usually reserved for python internal names

Reply:
Switched to single underscore.